about.html

```diff
@@ -0,0 +1,120 @@
+                      <li class="desklist2"><a href="#" class="desklink1">About</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para">"Welcome to VR!"</p>
+            <div class="clip">
+                <h1 class="heading1">VR EVOLUTION:<br>THE NEXT CHAPTER 2023</h1>
+            </div>
+            <div class="paraborder">
+                <p class="para1">We are excited to bring together professionals from across the VR industry for this two-day event exploring the latest trends, innovations, and challenges in VR technology. From keynote speeches and panel discussions to workshops, demonstrations, and networking events, we have designed an engaging and informative program that will provide attendees with valuable insights and practical skills.</p>
+            </div>
+            <div class="venue">
+                <p class="time">Please contact us per Email for any further questions<br>about VR EVO. THE NEXT CHAPTER 2023</p>
+                <p class="place">vrodyssey2023@evo.org</p>
+            </div>
+        </section>
+    </section>
+        <section class="challenge">
+            <div class="heading2">
+               <h2 class="heading3">VR Escape Room Challenge</h2>
+               <hr class="line">
+            </div>
+            <div class="program">
+                <img class="proimg" src="./images/escape.jpg">
+                <p class="propara">A VR escape room challenge where attendees work in teams to solve puzzles and escape a virtual environment within a set time frame.</p>  
+              </div>
+        </section>
+        <hr class="lineS">
+        <section class="past">
+            <h2 class="pasthead">See the past VR Odyssey Global Confrence</h2>
+            <hr class="line">
+            <p class="pastpara">Take a look at last two VR Odyssey Global Confrence took place in</p>
+            <div class="pastdiv">
+             <div id="bimg" class="pastimg">
```

Comment:
- [ ] Kindly 😃 try to make the order of design like the Behance as you see here there is a heading after that text after that image, so please try to add it like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225893159-5feec81c-0a4d-4b23-b49e-57315b9436fa.png)    |   ![image](https://user-images.githubusercontent.com/83360636/225893128-aeda2f40-5dc3-4b7a-920f-34991e05087f.png)  |